### PR TITLE
Direct domains

### PR DESCRIPTION
**Pull request by mmatczuk**

### CODE REVIEW

File: bind/flag.go [hunk 1]
```diff
@@ -126,8 +126,15 @@ func DenyDomains(fs *pflag.FlagSet, cfg *[]ruleset.RegexpListItem) {
...
 			"Deny requests to the specified domains. "+
-			"Prefix domains with '-' to exclude requests to certain domains from being denied.",
-	)
+			"Prefix domains with '-' to exclude requests to certain domains from being denied.")
+}
+
+func DirectDomains(fs *pflag.FlagSet, cfg *[]ruleset.RegexpListItem) {
+	fs.Var(anyflag.NewSliceValue[ruleset.RegexpListItem](*cfg, cfg, ruleset.ParseRegexpListItem),
+		"direct-domains", "[-]<regexp>,..."+
+			"Directly connect to the specified domains without using the upstream proxy. "+
```
Comment (Choraden) [inline bind/flag.go:135]: How about `Directly connect` -> `Connect directly`? It sounds odd to start with an adverb here.

**Review by Choraden — APPROVED** [review]
Everything looks good.

File: bind/flag.go (hunk 1 above)
Comment (mmatczuk) [reply bind/flag.go:135]: Yes.

**Comment by mmatczuk** [comment]
Fixes #306

**Pull request merged**